packages/firestore/src/local/local_serializer.ts

```diff
@@ -247,18 +245,9 @@ export class LocalSerializer {
       queryProto = this.remoteSerializer.toQueryTarget(targetData.target);
     }
 
-    let resumeToken: string;
-
-    if (targetData.resumeToken instanceof Uint8Array) {
-      // TODO(b/78771403): Convert tokens to strings during deserialization
-      assert(
-        SimpleDb.isMockPersistence(),
-        'Persisting non-string stream tokens is only supported with mock persistence .'
-      );
-      resumeToken = targetData.resumeToken.toString();
-    } else {
-      resumeToken = targetData.resumeToken;
-    }
+    // We can't store the resumeToken as a ByteString in IndexedDb, so we
```

Comment:
This is a much better comment than the ones in mutation_queue. The mutation queue comment explains what we do, but not really why we do it.